js/signup.js

```diff
@@ -0,0 +1,54 @@
+import {  authValidity, emailInput, passwordInput, displayError, removeError, validateEmail, validatePassword, psVisibility } from "./auth.js";
+
+emailInput.addEventListener("focusout", validateEmail);
+passwordInput.addEventListener("focusout",validatePassword);
+
+const ps_visibility = document.getElementsByClassName("ps-visibility");
+Array.from(ps_visibility).forEach(element => {
+    element.addEventListener("click", psVisibility);
+});
```

Comment:
💬 여담
순회하고 싶으실때 아래처럼하셔도 됩니다~

```suggestion
  const psVisibilityElements = document.querySelectorAll(".ps-visibility");
    
  psVisibilityElements.forEach((element) => {
    element.addEventListener("click", psVisibility);
  });
```